README.md

```diff
@@ -172,8 +172,18 @@ User permissions according to a type of user is planned to become available in a
 |user04 | Pre-clinical-intake user
 |user07 | Bioinformatics-section user
 
-![Login page](./docs/figures/login.png "Login page")
-![User logged in](./docs/figures/logged-in.png "Developer user logged in")
+![Rosalution logins](./docs/figures/rosalution-logins.png "Rosalution logins")
+
+### Adding Cases
+
+Users can create new cases on Rosalution by clicking on the '+' card to add a new case, as shown in the below figures.
+Doing this will prompt a dialog to import files to create a new Rosalution case. These files must be in .json format.
+Once uploaded, a 'Successful import' dialog will be displayed.
```

Comment:
```suggestion
Once uploaded, a 'Successful import' dialog displays.
```

Reply:
Thanks for pointing this out! Added suggestion.